### PR TITLE
feat(eventing): add supportability for nats

### PR DESCRIPTION
**Pull request by datacore-vvarakantham**
Added supportability for NATS service.

## Description

Adding NATS  pods' logs to the support bundle.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.

### CODE REVIEW

File: chart/README.md [hunk 1]
```diff
@@ -191,3 +191,55 @@ $ helm install my-release openebs/mayastor
 | priorityClassName | Pod scheduling priority. Setting this value will apply to all components except the external Chart dependencies. If any component has `priorityClassName` set, then this value would be overridden for that component. For external components like etcd, jaeger or loki-stack, PriorityClass can only be set at component level. | `""` |
 | tolerations | Tolerations to be applied to all components except external Chart dependencies. If any component has tolerations set, then it would override this value. For external components like etcd, jaeger and loki-stack, tolerations can only be set at component level. | `[]` |
 
+# nats
```
Comment (tiagolobocastro) [inline chart/README.md:194]: Why is this being included here?

Comment (datacore-vvarakantham) [reply chart/README.md:194]: This whole thing is being added automatically by the pre-commit.

**Review by niladrih — APPROVED** [review]
LGTM. README changes are not needed.

**Comment by niladrih** [comment]
@datacore-vvarakantham @tiagolobocastro -- the script is adding the nats documentation because the nats dependency chart tar is unpacked in the chart/charts directory.

@datacore-vvarakantham could you remove the chart/charts directory and try again.

File: chart/README.md (hunk 1 above)
Comment (niladrih) [reply chart/README.md:194]: The script is picking up on the chart files of the nats subchart. The subchart was likely pulled in with a `dependency update`, and unpacked in-place.

Removing the subchart dir should generate the README.md correctly, I think.

Comment (datacore-vvarakantham) [reply chart/README.md]: Yes, removing the chart/charts directory works.

**Comment by tiagolobocastro** [comment]
> @datacore-vvarakantham @tiagolobocastro -- the script is adding the nats documentation because the nats dependency chart tar is unpacked in the chart/charts directory.

Why is this only happening now?

**Comment by datacore-vvarakantham** [comment]
> > @datacore-vvarakantham @tiagolobocastro -- the script is adding the nats documentation because the nats dependency chart tar is unpacked in the chart/charts directory.
> 
> Why is this only happening now?

I have unpacked to check some values actually. That was why it might be adding the doc

**Review by tiagolobocastro — APPROVED** [review]

**Comment by datacore-vvarakantham** [comment]
bors merge

**Comment by datacore-vvarakantham** [comment]
bors merge

**Comment by niladrih** [comment]
This fixes the helm-docs issue: https://github.com/openebs/mayastor-extensions/pull/435.

**Pull request merged**